backend/jobsearch/src/main/kotlin/com/jobsearch/dto/VacancyResponseDTO.kt

```diff
@@ -0,0 +1,12 @@
+package com.jobsearch.dto
+data class VacancyResponseDTO(
+    val id: Int,
+    var name: String,
+    val companyName: String,
+    val salaryExpectation: Int,
+    val yearsOfExperience: Int,
+    val description: String,
+    val jobFamilyId: Int,
+    val jobFamilyName: String,
+    val managerId: Int?
```

Comment:
If I'm not mistaken the vacancy could be created by a manager only
So, `managerId` should not be nullable